interactions/client/bot.py

```diff
@@ -1416,6 +1421,33 @@ async def modify(
 
         return User(**data)
 
+    async def get(self, obj: Type[_T], **kwargs) -> _T:
+        """
+        A helper method for retrieving data from the Discord API in its object representation.
+
+        :param obj: The object to get. Should be a class object (not an instance!). For example: `interactions.Channel`.
+        :type obj: object
+        :param **kwargs: The arguments to pass to the HTTP method.
+        :type **kwargs: dict
+        :return: The object we're trying to get.
+        :rtype: object
+        """
+
+        if isinstance(obj, (Channel, Emoji, Guild, Member, Message, Role, Sticker, User)):
+            raise TypeError("The object must not be an instance of a class!")
+
+        _name = f"get_{obj.__name__.lower()}"
+
+        if obj in (Role, Emoji):
+            _guild = Guild(**await self._http.get_guild(kwargs.pop("guild_id")), _client=self._http)
+            _func = getattr(_guild, _name)
+            return await _func(**kwargs)
+
+        else:
+            _func = getattr(self._http, _name)
+            _obj = await _func(**kwargs)
+            return obj(**_obj, _client=self._http)
```

Comment:
You could technically remove the else condition if you want:
```suggestion
        _func = getattr(self._http, _name)
        _obj = await _func(**kwargs)
        return obj(**_obj, _client=self._http)
```